lib/constructs/interface.js

```diff
@@ -1444,12 +1487,39 @@ class Interface {
           globalObject[ctorRegistrySymbol] = Object.create(null);
         }
         globalObject[ctorRegistrySymbol][interfaceName] = ${name};
+    `;
 
+    if (!utils.getExtAttr(this.idl.extAttrs, "NoInterfaceObject")) {
+      this.str += `
         Object.defineProperty(globalObject, interfaceName, {
           configurable: true,
           writable: true,
           value: ${name}
         });
+      `;
+
+      if (this.legacyWindowAliases) {
+        this.str += `
+        if (globalName === "Window") {
```

Comment:
```suggestion
          if (globalName === "Window") {
```

The existing convention is to indent the template literal by at least one tab level.

Reply:
I matched the indentation of the other template literals.